src/subwindows/stereoCameraSettingsDialog.cpp

```diff
@@ -759,18 +894,15 @@ void StereoCameraSettingsDialog::onSetImageROIoffsetY(int val) {
     updateCamImageRegionsWidget();
 }
 
-void StereoCameraSettingsDialog::updateImageROISettingsMin(int binningVal) {
-    imageROIwidthInputBox->setMinimum(minImageSize/binningVal);
-    imageROIwidthInputBox->setSingleStep(imageSizeChangeSingleStep/binningVal);
-    imageROIheightInputBox->setMinimum(minImageSize/binningVal);
-    imageROIheightInputBox->setSingleStep(imageSizeChangeSingleStep/binningVal);
-    imageROIoffsetXInputBox->setMinimum(0);
-    imageROIoffsetXInputBox->setSingleStep(imageSizeChangeSingleStep/binningVal);
-    imageROIoffsetYInputBox->setMinimum(0);
-    imageROIoffsetYInputBox->setSingleStep(imageSizeChangeSingleStep/binningVal);
-}
-
 void StereoCameraSettingsDialog::updateImageROISettingsMax() {
+    if(!camera->isOpen()) {
+        imageROIwidthInputBox->setMaximum(std::numeric_limits<short>::max());
```

Comment:
Why are these necessary?

Reply:
The cameras can be opened only after the settings dialog is drawn, in a different step, and setting the queried max value (which is 0 in case of a closed camera) when the camera is closed could cause problems later, when upon opening the camera the actual values are to be filled there, before the real maximum is set. The max of short type is just a good arbitrary maximum when we do not know the real maximum